ci/scale-run.sh

```diff
@@ -15,6 +15,7 @@ pushd $OVN_SCALE_TOP
 cd ansible/docker
 make ovsrepo=$OVS_REPO ovsbranch=$OVS_BRANCH
 popd
+sudo docker images
```

Comment:
@mestery thanks and will fix. BTW, when I change this, shall I also replace the sudo in [this line](https://github.com/openvswitch/ovn-scale-test/blob/master/ci/scale-run.sh#L24)?